packages/react/src/components/Checkbox/Checkbox-test.js

```diff
@@ -190,3 +193,35 @@ describe('CheckboxSkeleton', () => {
     });
   });
 });
+
+describe('Checkbox accessibility', () => {
+  afterEach(cleanup);
+
+  it('should have no Axe violationk', async () => {
+    render(<Checkbox labelText="Checkbox label" id="test_id" />);
+    await expect(screen.getByText('Checkbox label')).toHaveNoAxeViolations();
```

Comment:
Will this check the whole checkbox component or the node that this text is inside? (Or does this selector get both? lol)

Reply:
ah! great catch. I was using the wrong selector query there and it was returning _just_ the span with that text in it 🤦🏽 

`.getByLabelText()` is what I wanted here which returns
![image](https://user-images.githubusercontent.com/40970507/101541879-98a07780-3967-11eb-961e-c8a5ef1c922a.png)